server/sonar-server/src/main/java/org/sonar/server/license/ws/ListAction.java

```diff
@@ -33,19 +35,20 @@
 import org.sonar.api.server.ws.Request;
 import org.sonar.api.server.ws.Response;
 import org.sonar.api.server.ws.WebService;
+import org.sonar.core.util.stream.Collectors;
 import org.sonar.db.DbClient;
 import org.sonar.db.DbSession;
-import org.sonar.db.property.PropertyDto;
+import org.sonar.server.setting.ws.Setting;
+import org.sonar.server.setting.ws.SettingsFinder;
 import org.sonar.server.user.UserSession;
 import org.sonar.server.ws.WsAction;
 import org.sonarqube.ws.Licenses;
 import org.sonarqube.ws.Licenses.ListWsResponse;
 
-import static com.google.common.collect.Sets.newHashSet;
+import static org.elasticsearch.common.Strings.isNullOrEmpty;
```

Comment:
use the Guava version, I suggest to remove this package from IntelliJ imports, ping me if you don't know how to do it. Very useful